datahub-web-react/src/app/analyticsDashboard/components/TimeSeriesChart.tsx

```diff
@@ -42,39 +42,17 @@ function computeLines(chartData: TimeSeriesChartType, insertBlankPoints: boolean
 
     const startDate = new Date(Number(chartData.dateRange.start));
     const endDate = new Date(Number(chartData.dateRange.end));
-    const intervalMs = INTERVAL_TO_SECONDS[chartData.interval] * 1000;
     const returnLines: NamedLine[] = [];
     chartData.lines.forEach((line) => {
         const newLine = [...line.data];
-        for (let i = endDate.getTime(); i > startDate.getTime(); i -= intervalMs) {
+        for (let i = startDate; i <= endDate; i = addInterval(1, i, chartData.interval)) {
             const pointOverlap = line.data.filter((point) => {
-                return Math.abs(new Date(point.x).getTime() - i) > intervalMs;
+                return Math.abs(new Date(point.x).getTime() - i.getTime()) === 0;
             });
-            if (pointOverlap) {
-                break;
-            }
-
-            const pointGreater = line.data.find((point) => new Date(point.x).getTime() > i);
-            if (pointGreater) {
-                break;
-            }
-            insertBlankAt(i, newLine);
```

Comment:
This is so that we have 0 values where we do not have data - I'm assuming this was addressed on the server